mrblib/global.rb

```diff
@@ -8,4 +8,5 @@
 #
 
 RUBY_VERSION = "1.9"
+MRUBY_VERSION = "2.1.1"
```

Comment:
You can specify mruby version here.
It is actually a name of tag/branch of mruby repository.